.travis.yml

```diff
@@ -1,14 +1,27 @@
 # Travis CI (https://travis-ci.org/)
+# This will only work on your repo if you have an account on travis and you
+# have set it up to run continuous integration on this this repo
 
-language: c
+# Linux distribution (bionic beaver)
 dist: bionic
+
+# Language and version
+language: python
+python:
+  - "3.6" # current default Python on Travis CI
+
 cache:
   apt: true # only works with Pro version
 
+# Install the miss_hit linter
 before_install:
-  - cd .. && git clone https://github.com/florianschanda/miss_hit.git && export PATH=$PATH:`pwd`/miss_hit && cd CPP_PTB
+  - pip3 install miss_hit
```

Comment:
🎉

Reply:
:rofl: 

yeah it was pretty simple in fact. 

set up travis so the default language is python and then pip install...